Dockerfile

```diff
@@ -24,6 +24,13 @@ RUN --mount=type=cache,target=/var/lib/apt/lists --mount=target=/var/cache/apt,t
       docker-buildx-plugin \
     && docker --version
 
+# PSQL
+RUN --mount=type=cache,target=/var/lib/apt/lists --mount=target=/var/cache/apt,type=cache \
```

Comment:
what makes this necessary now?

Reply:
Have been installing PSQL often locally and in AWS.  This removes the need.